examples/jsm/loaders/VOXLoader.d.ts

```diff
@@ -0,0 +1,13 @@
+import {
+	Loader,
+	LoadingManager
+} from '../../../src/Three';
+
+export class VOXLoader extends Loader {
+
+	constructor( manager?: LoadingManager );
+
+	load( url: string, onLoad: ( geometry: BufferGeometry ) => void, onProgress?: ( event: ProgressEvent ) => void, onError?: ( event: ErrorEvent ) => void ): void;
```

Comment:
The onLoad parameter is not really BufferGeometry, it looks like?

Reply:
Yep! Will fix.
Returning an object for now as the data can be used in different ways.

Reply:
Fixed: https://github.com/mrdoob/three.js/commit/804a16607746a35d3e2d6fa4ce5e30dfb463fdbd